content/xr_design_and_accessibility.mdx

```diff
@@ -0,0 +1,53 @@
+| Location | <li>At a fixed distance in front of the user</li> <li>On a fixed location</li> <li>Above a character’s head</li> | What does the text convey and what location will best affect the user’s experience? |
+| Font size | <li>Small</li><li>Medium</li><li>Large</li> | Set the default size to medium and allow the user to adjust font size (either by creating preset options or scale) to their comfort level.  This should start at 10% of the screen’s height and adjust as needed. |
+| Friendly fonts | <li>Arial</li> <li>OpenDyslexic</li> | Sans serif and monospaced fonts are often easier to read. Allowing a user to choose a preferred font based on their needs means that their experience is less likely to be diminished by something that we (the devs) can easily adjust. |
+| Visual indicators | <li>Arrows</li> <li>Animations</li> | Due to the nature of the 360 degree experience, we often lack control over what the user is seeing. If there is text that needs to be read, cue the user with visual indicators if they’re looking in the wrong direction. |
+| Alignment | Left-aligned and no justification | Left-aligned text with no justification is easier to read for those with dyslexia. |
+
+# Subtitles
+While most of the recommendations for subtitles will fall under the text category, consider any audio and the cues that might be missed for a person with a hearing disability or low volume settings. Is your experience the same without audio entirely? Would it benefit from captioning?
+
+# Display
+| | What to Consider |
+|:-----|-----:|
+| Color | Is color being used to indicate important behavior, like red for “stop” or green for “go”? Consider supplementing these behaviors with patterns, symbols or shapes to enforce differences. |
+| Brightness | Brightness presets can assist people with low vision. |
+| Contrast | Contrast presets can assist people with low vision. Verify if a change in contrast still offers the same amount of depth. |
+| Distance | Consider allowing functionality to bring objects closer. This can help people with low vision or limited mobility. |
+| Size | Size presets can include changing the size of fonts, objects, buttons, etc. Some people would benefit from interacting with a bigger hitbox. |
+
+# Inputs
+Input alternatives can greatly affect those with limited mobility, whether it is situation, temporary or permanent.
+
+|| <div style="width:300px">Alternatives</div> |
+|:-----|-----:|
+| Head movement | Directional input (moving the joystick left or right) <li>Quick turns (rotating 30-45 degrees)</li> <li>Sharp turns (rotating 30-45 degrees with no transition</li> Audio commands |
+| Locomotion | <li>Teleportation</li> <li>Free locomotion (free movement within one’s playspace)</li> <li>Rails (moves through the experience for the user. Can cause nausea if emulating real speed)</li> |
```

Comment:
`Rails (moves through the experience for the user. Can cause nausea if emulating real speed)`

Would read better as:

`Rails (moves through the experience for the user; can cause nausea if emulating real speed)`